test/cookie_coder_test.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "test_helper"
+
+describe Identity::CookieCoder do
+  secret = "Pw/Xv58CsV8FVUQldtbHRZBoGqv8pJ1u55JdlhsRp9A="
+  another_secret = "r1auRylYrR9WM3dhRkfXRwQ5nxcNGR2lBkCLgXl94gU="
+
+  it "encryptes a cookie, then descripts a cookie" do
```

Comment:
Fixed in https://github.com/heroku/identity/commit/4bab7dc3da47f527cf4e79e0e16e11a022f74e53.